test/python/test_updateconfig.py

```diff
@@ -23,9 +23,6 @@
 
 class TestUpdateConfig(NgraphTest):
 
-    @pytest.mark.skipif(
```

Comment:
With the decorator removed it would be executed with rewrite pass, so let's update this too work with new grappler API